testing/bdd/features/unit_conversion/ebuttd_line_height_conversion.feature

```diff
@@ -0,0 +1,21 @@
+Feature: EBU-TT-D lineHeight conversion
+
+    On converting to EBU-TT-D, lineHeight values must be translated into percentages.
+
+    Examples:
+        | xml_file                     |
+        | style_element_references.xml |
+
+    Scenario: The value of tts:lineHeight shall be recalculated and expressed in percentage
+        Given an xml file <xml_file>
+        When it contains style "s1"
+        And style "s1" has attribute "lineHeight" set to <lineHeight>
+        And style "s1" has attribute "fontSize" set to <fontSize>
+        And it contains some text with style "s1"
+        When the document is generated
+        And the EBU-TT-Live document is converted to EBU-TT-D
+        Then the ebu_tt_d document contains style "autogenFontStyle_None_200.0" with attribute "lineHeight" set to <ebu_tt_d_value>
+
+        Examples:
+            | lineHeight | fontSize | ebu_tt_d_value |
+            | 2c         | 2c       | 100.0%         |
```

Comment:
I'd like to see more permutations here. Why does the `ebu_tt_d_value` have a decimal place?

Reply:
I remember trying to add more examples and what they returned didn't make sense? Something like putting 1c for both giving us no ebu_tt_d value at all? I think it was related to the autogen font style code
We tried it without a decimal place and it didn't pass, because 100% != 100.0%, not sure why the code gives it a decimal place though

Reply:
Fixed in 954751c which adds more valid lineHeight values including `normal` and also fixes the decimal places problem.